docs/devguide/hexformat.rst

```diff
@@ -120,23 +120,12 @@ found here: `microbit-foundation/micropython-microbit-v2 <https://github.com/mic
 This is a port of MicroPython to the micro:bit which uses CODAL as the 
```

Comment:
Let's also change this page title from `Firmware Hex File` to `Micropython Hex File`.

Reply:
Done.